app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,83 @@
+class RecipesController < ApplicationController
+  load_and_authorize_resource
+  before_action :set_recipe, only: %i[show edit update destroy]
+
+  # GET /recipes or /recipes.json
+  def index
+    @recipes = Recipe.all
```

Comment:
* As stated in the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Make%20sure%20there%20are%20no%20N%2B1%20queries%20happening.) n + 1 issues should be handled , Kindly fix all n +1 issues 🙏

Reply:
@Davidosky007 thanks for the review
I have fix the problem